xdr2/bench_test.go

```diff
@@ -14,14 +14,12 @@
  * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-package xdr_test
+package xdr
```

Comment:
The point is that this style of testing makes forking impossible, because of the absolute import path used to import the `xdr2` package. And unfortunately, you can't use relative imports in this context.

So for instance, the CI would be unable to test this pull request because it would use the upstream version of the `xdr2` package, rather than the version in this branch.